playground/mocks/config/responseConfig.js

```diff
@@ -1047,5 +1047,5 @@ const selectOktaVerifyMethod = {
 };
 
 module.exports = {
-  mocks: idx
+  mocks: Test.EnrollAuthenticatorWebAuthn.mock
```

Comment:
Is this on purpose?

Reply:
will be removed before merge. I need to promote auth-js and update/test again. I'll clean this up too